mobile/node.go

```diff
@@ -82,7 +82,7 @@ func NewNode(repoPath string, authenticationToken string, testnet bool, userAgen
 }
 
 // NewNodeWithConfig create a new node using the configuration file from NewNode()
-func NewNodeWithConfig(config NodeConfig, password string, mnemonic string) (*Node, error) {
+func NewNodeWithConfig(config *NodeConfig, password string, mnemonic string) (*Node, error) {
```

Comment:
ARGH! 👏 @tyler-smith  I was so close! Was trying to decipher that error message @drwasho posted but I didn't have gomobile setup to test. Thanks for this!